news/_posts/core-weekly/2020-02-18-coreweekly-week-07-2020.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 7 of 2020"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2020-02-18
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 10th to Sunday 16th of February 2020.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+Last week our Product Team had a wonderful surprise: they received a notification telling them someone opened a [Pull Request on the Specifications repository](https://github.com/PrestaShop/prestashop-specs/pull/86) ! They were delighted to see the community willing to collaborate on us on this project.
```

Comment:
@sam-pires @colinegin Please tell me if you agree with this 😉

Reply:
A few comments @matks : 
- i'm not sure people are aware of this repo, so maybe we should give a few info about it
- i'm not sure we should call it a "project"
- maybe we could directly thank the contributor whose name is arouiadib

Reply:
All right, can you write what you'd like to say 😉 ?

I dont want to write something, then you provide feedback, then I write something, then you provide feedback ... 😅

Reply:
My proposition : "Last week our Product Team had a wonderful surprise: they received a notification telling them someone opened a [Pull Request on the Specifications repository](https://github.com/PrestaShop/prestashop-specs/pull/86) ! This is the first community contribution since this repo dedicated to centralize all core and module functional specifications has been created. They were delighted to see the community willing to collaborate with us on this huge work."

Wdyt @matks @sam-pires ?

Reply:
I ❤️ it and included it here 😉 https://github.com/PrestaShop/prestashop.github.io/pull/647/files#diff-1e48a63d130a67f0c0cbb5a09ea0c18aR21

Reply:
👍 
I'll quickly clean the repo tree we messed up with Gitbook, and add some guidelines on how to contribute.

Reply:
Can you wait a bit before merging? 🙂  like a couple of hours

Reply:
So we're good, I can publish ? 😄